<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>time_register_widget</class>
 <widget class="QWidget" name="time_register_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>205</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Calibri</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Run time register</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="1,7,2,1">
   <item row="0" column="0">
    <widget class="QLabel" name="lbl_run_number">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>32</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="accessibleName">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>23</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="margin">
      <number>6</number>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="lbl_category_name">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Category name</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="margin">
      <number>4</number>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="lbl_first_name">
     <property name="text">
      <string>---</string>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <property name="indent">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../pix/n2.png</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="lbl_second_name">
     <property name="text">
      <string>---</string>
     </property>
     <property name="indent">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_6">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../pix/n3.png</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../pix/n1.png</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="lbl_third_name">
     <property name="text">
      <string>---</string>
     </property>
     <property name="indent">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_7">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../pix/n4.png</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="lbl_fourth_name">
     <property name="text">
      <string>---</string>
     </property>
     <property name="indent">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QComboBox" name="cb_second_status"/>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Time</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Result</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QComboBox" name="cb_first_status"/>
   </item>
   <item row="4" column="3">
    <widget class="QComboBox" name="cb_third_status"/>
   </item>
   <item row="5" column="3">
    <widget class="QComboBox" name="cb_fourth_status"/>
   </item>
   <item row="7" column="3">
    <widget class="QPushButton" name="btn_register_time">
     <property name="text">
      <string>Done</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="Line" name="line">
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="Line" name="line_3">
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="Line" name="line_4">
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="Line" name="line_2">
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="inp_first_time">
     <property name="inputMask">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="Line" name="line_5">
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="Line" name="line_6">
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="Line" name="line_7">
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="Line" name="line_8">
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="inp_second_time">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLineEdit" name="inp_third_time">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLineEdit" name="inp_fourth_time">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QPushButton" name="btn_delete_run">
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="text">
      <string>Delete run</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>inp_first_time</tabstop>
  <tabstop>inp_second_time</tabstop>
  <tabstop>inp_third_time</tabstop>
  <tabstop>inp_fourth_time</tabstop>
  <tabstop>btn_register_time</tabstop>
  <tabstop>cb_first_status</tabstop>
  <tabstop>cb_second_status</tabstop>
  <tabstop>cb_third_status</tabstop>
  <tabstop>cb_fourth_status</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
